### PR TITLE
Add snapshot and job ci test and remove the mock dir

### PR DESCRIPTION
**Pull request by laura-ding**
depends on https://github.com/vesoft-inc/nebula-storage/pull/138

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

File: src/mock/test/BalanceTest.cpp [hunk 1]
```diff
@@ -1,182 +0,0 @@
-/* Copyright (c) 2020 vesoft inc. All rights reserved.
```
Comment (Shylock-Hg) [inline src/mock/test/BalanceTest.cpp:1]: This need tests too.

**Review by yixinglu — COMMENTED** [review]
Please solve the conflicts and suggest to rename `single_execute` folder to `job`

File: src/mock/test/BalanceTest.cpp (hunk 1 above)
Comment (laura-ding) [reply src/mock/test/BalanceTest.cpp:1]: The test environment is now not clustered, The balancer testing is meaningless.

File: ci/test.sh [hunk 2]
```diff
@@ -103,7 +104,16 @@ function run_test() {
...
         $PROJ_DIR/tests/query/stateless/test_simple_query.py \
         $PROJ_DIR/tests/query/stateless/test_keyword.py \
-        $PROJ_DIR/tests/query/stateless/test_lookup.py
+        $PROJ_DIR/tests/query/stateless/test_lookup.py &
+
+    for pid in $(jobs -p)
+    do
+        wait $pid
+        status=$?
+        if [ $status != 0 ];then
```
Comment (Shylock-Hg) [inline ci/test.sh:113]: `if [ ! wait $pid] ` directly

File: ci/test.sh [hunk 3]
```diff
@@ -103,7 +104,16 @@ function run_test() {
...
+            exit 1
```
Comment (Shylock-Hg) [inline ci/test.sh:114]: `exit status` better.

**Review by yixinglu — APPROVED** [review]

File: ci/test.sh [hunk 4]
```diff
@@ -110,6 +122,8 @@ function test_in_cluster() {
         --debug_log=false \
         --address="nebulaclusters-graphd:3699" \
         $testpath
+
+    ./ntr --debug_log=false $PROJ_DIR/tests/job/*
```
Comment (yixinglu) [inline ci/test.sh:126]: `--address="nebulaclusters-graphd:3699"`

**Review by yixinglu — APPROVED** [review]

**Review by jievince — APPROVED** [review]
LGTM

**Pull request merged**